osquery/tables/networking/utils.h

```diff
@@ -31,5 +34,34 @@ std::string ipAsString(const struct in_addr *in);
 std::string macAsString(const struct ifaddrs *addr);
 std::string macAsString(const char *addr);
 int netmaskFromIP(const struct sockaddr *in);
+
+// Linux proc protocol define to net stats file name.
+extern const std::map<int, std::string> kLinuxProtocolNames;
+// A map of socket handles (inodes) to their pid and file descriptor.
+typedef std::map<std::string, std::pair<std::string, std::string>> InodeMap;
+std::string addressFromHex(const std::string& encoded_address, int family);
+unsigned short portFromHex(const std::string& encoded_port);
```

Comment:
I know we haven't set a good example in this header but do you mind include spaces between the symbol declarations. And if possible using the `///` short-doc docblock if it doesn't need param and return value documentation?